spec/spec_helper.rb

```diff
@@ -2,13 +2,21 @@
 
 if ENV['DISABLE_SIMPLECOV'] != 'true'
   require 'simplecov'
+  require 'simplecov-lcov'
+  SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+  SimpleCov.formatter = SimpleCov::Formatter::LcovFormatter
   SimpleCov.start 'rails' do
+    enable_coverage :branch
+    enable_coverage_for_eval
```

Comment:
Enabling this eval coverage causes some (not fixable) error output when running - https://github.com/simplecov-ruby/simplecov/issues/1057 -- the report stays accurate, but there's warnings that you can't actually do anything about (as described in the issue: from erb/haml/etc having an intermediate step in their template processes)